dags/suggestions/business_logic/db_write_validsuggestions.py

```diff
@@ -0,0 +1,172 @@
+import logging
+
+from shared.tasks.database_logic.db_manager import PostgresConnectionManager
+from sources.config import shared_constants as constants
+from utils import logging_utils as log
+
+
+def db_write_validsuggestions(data_from_db: dict):
```

Comment:
```suggestion
def db_write_validsuggestions(dfs_normalized_acteur_update: dict):
```

Voir pour respecter la signature de la fonction qui arrive à ce dict